### PR TITLE
Setup Testing Scripts and Refactor some code with linters

### PR DESCRIPTION
**Pull request by shrinathprabhu**
Resolves [AR-1957](https://team-1624093970686.atlassian.net/browse/AR-1957).

## Changes

- [x] Add unit test scripts
- [x] Add e2e test scripts

- Added .vscode setting for environment variables (was suggested/prompted by the editor)
- Added cypress related files (cypress.json, cypress/plugins, cypress/support)
- Added test scripts in package.json
- Replaced exports in `constants.ts` as suggested by linter
- In `components/app-configure/AppChain.vue`, linter was run because of changes (related to constants.ts) and it replaced (template, style, script) to (script, template, style) format
- Ran linter on `main.ts`, `router/index.ts`, `services/get-env-api.ts`, `utils/constants.ts` and `components/app-configure/AppChain.vue` because of changes required in `utils/constants.ts`

## Checklist

- [x] The branch name follows the format: `developer/AR-XXX-issue-name`.
- [x] The changes have been tested locally.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -15,7 +15,14 @@
     "lint:all:markdownlint": "npm run lint:markdownlint -- \"**/*.md\" --ignore-path .gitignore",
     "lint:all:prettier": "npm run lint:prettier -- \"**/*.{ts,json,css,scss,vue,html,md}\" --ignore-path .gitignore",
     "lint:all:stylelint": "npm run lint:stylelint -- \"**/*.{vue,css,scss}\" --ignore-path .gitignore",
-    "prepare": "husky install"
+    "prepare": "husky install",
+    "test:e2e": "start-server-and-test preview :5050 'cypress open'",
+    "test:e2e:ci": "start-server-and-test preview :5050 'cypress run'",
+    "cypress": "cypress open",
```
Comment (ajithranka) [inline package.json:21]: Do we need this script? How does it get used in the testing workflow?

```
    "cypress": "cypress open",
```

File: src/utils/constants.ts [hunk 2]
```diff
@@ -1,38 +1,42 @@
...
+const constants = {
   sentry,
   api,
   arcanaAppId,
   isAppDown,
   chains,
-};
+}
+
+console.log(import.meta.env)
```
Comment (ajithranka) [inline src/utils/constants.ts:40]: Stray console.log.

File: .vscode/settings.json [hunk 3]
```diff
@@ -44,5 +44,8 @@
   "[markdown]": {
     "editor.wordWrap": "wordWrapColumn",
     "editor.wordWrapColumn": 80
+  },
+  "[dotenv]": {
+    "editor.defaultFormatter": "foxundermoon.shell-format"
```
Comment (ajithranka) [inline .vscode/settings.json:49]: Not very keen on adding more extensions to the shared settings. You can have always have more extensions active than the ones listed out in `extensions.json`.

If you feel strongly that we need to include this, please add it to the `extensions.json` file.

File: cypress.json [hunk 4]
```diff
@@ -0,0 +1 @@
+{}
```
Comment (ajithranka) [inline cypress.json:1]: From the Preamble template:

```
{
  "baseUrl": "http://localhost:5050"
}
```

File: src/components/app-configure/AppChain.vue [hunk 5]
```diff
@@ -1,21 +1,99 @@
+<script lang="ts">
```
Comment (ajithranka) [inline src/components/app-configure/AppChain.vue:1]: Can we use the `script setup` syntax here?

**Review by ajithranka — CHANGES_REQUESTED** [review]

File: src/components/app-configure/AppChain.vue (hunk 5 above)
Comment (shrinathprabhu) [reply src/components/app-configure/AppChain.vue:1]: We'll need to do changes for that. Once I'm done with Configure redesign, I'll work on migrating things to `script setup` syntax. Configure redesign will use script setup syntax

File: .vscode/settings.json (hunk 3 above)
Comment (shrinathprabhu) [reply .vscode/settings.json:49]: Understood

File: package.json (hunk 1 above)
Comment (shrinathprabhu) [reply package.json:21]: No, I was just testing why cypress was not opening. I forgot to remove it.

**Review by ajithranka — APPROVED** [review]

**Review by karthik-durai — APPROVED** [review]

**Pull request merged**